### PR TITLE
Add crash reporter opt in setting and UI

### PR DESCRIPTION
**Pull request by mkestler-rtp**
Add a setting check to enable autosend of crash reports.  Also add a checkbox on the crash reporter dialog to enable it there as well.

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -317,6 +317,9 @@ settings:
     value: Tunnel VPN through port 53/DNS
     comment: Tunnel is used as a verb, as in to direct traffic through
   tunnelPort53Description: Use port 53/DNS for Wireguard traffic, which may help evade firewalls
+  crashOptInLabel:
+    value: Autosend crash reports
+    comment: Main label for the auto send crash reports option.  Goes above the longer description.
```
Comment (flodolo) [inline translations/strings.yaml:322]:
```suggestion
    comment: Main label for the autosend crash reports option. Goes above the longer description.
```

Let's avoid double spaces everywhere.

**Review by flodolo — APPROVED** [review]

File: src/ui/settings/ViewPrivacySecurity.qml [hunk 2]
```diff
@@ -55,19 +55,14 @@ Item {
...
-
             VPNCheckBoxRow {
                 id: dataCollection
                 objectName: "dataCollection"
                 width: parent.width - VPNTheme.theme.windowMargin
                 anchors.left: parent.left
                 anchors.right: parent.right
                 anchors.rightMargin: VPNTheme.theme.windowMargin
-
+                showDivider: false
```
Comment (lesleyjanenorton) [inline src/ui/settings/ViewPrivacySecurity.qml:65]: Similar question as below but if `crashOptInCheckBox` is only visible on certain platforms, then this should be visible if `!crashOptInCheckBox.visible`.

Comment (mkestler-mozilla) [reply src/ui/settings/ViewPrivacySecurity.qml:65]: Data collection is for glean, not crash reporting.  It is available everywhere.

**Review by lesleyjanenorton — APPROVED** [review]

File: src/ui/settings/ViewPrivacySecurity.qml (hunk 2 above)
Comment (mkestler-mozilla) [reply src/ui/settings/ViewPrivacySecurity.qml]: I've updated this now that I understand the comments.

**Comment by bakulf** [comment]
@lesleyjanenorton do you think we still need this PR? It has never landed

**Comment by lesleyjanenorton** [comment]
@bakulf not sure! It seems like nice to have but do you remember if there was some reason why it wasn't merged before? I can rebase tomorrow and try running.

**Comment by birdsarah** [comment]
Closing as stale.

**Pull request closed without merging**